README.md

```diff
@@ -6,4 +6,71 @@
 
 ## Overview
 
-UNDER CONSTRUCTION
+Utility package that provides end user friendly methods for feature scalings and polynomial
+basis expansion.
+
+### Standardization
+Standardization of data sets result in variables with a mean of 0 and variance of 1.
+A common use case would be to fit a `StandardScaler` to the training data and later
+apply the same transformation to the test data. `StandardScaler` is used with the 
+functions `fit()`, `transform()` and `fit_transform()` as shown below.
+
+```julia
+
+    fit(StandardScaler, X[, μ, σ; obsdim, operate_on])
+
+    fit_transform(StandardScaler, X[, μ, σ; obsdim, operate_on])
+```
+
+`X`         :  Data of type Matrix or `DataFrame`.
+
+`μ`         :  Vector or scalar describing the translation.
+               Defaults to mean(X, obsdim)
+
+`σ`         :  Vector or scalar describing the scale.
+               Defaults to std(X, obsdim)
+
+`obsdim`    :  Specify which axis corresponds to observations.
+               Defaults to obsdim=2 (observations are columns of matrix)
+               For DataFrames `obsdim` is obsolete and rescaling occurs
+               column wise.
+
+`operate_on`:  Specify the indices of columns or rows to be centered.
+               Defaults to all columns/rows.
+               For DataFrames this must be a vector of symbols, not indices.
+               E.g. `operate_on`=[1,3] will perform centering on columns
+               with index 1 and 3 only (if obsdim=1, else rows 1 and 3)
+
+Note on DataFrames:
+Columns containing `NA` values are skipped.
+Columns containing non numeric elements are skipped.
+
+Examples:
+
+
+    Xtrain = rand(100, 4)
```

Comment:
maybe use a julia codeblock explicitly so that the readme has syntax highlighting